llvm/lib/Transforms/InstCombine/InstCombineCalls.cpp

```diff
@@ -3364,6 +3421,12 @@ Instruction *InstCombinerImpl::visitCallInst(CallInst &CI) {
       //   zext(vector_reduce_{and,or}(<n x i1>))
       Value *Arg = II->getArgOperand(0);
       Value *Vect;
+
+      if (Value *NewOp = simplifyReductionOperand(Arg, true)) {
```

Comment:
Nit: Here and above, style is `/*CanReorderLanes=*/true`

Reply:
Done!